### ISSUE TITLE
Fail if specifying non-existing test-suite

### ISSUE THREAD

**Issue by jordansamuels** [evansde77/cirrus#159]
If the user does something like `git cirrus test --suite=non_existent` then we can get errors that look like:
```
Fatal error: local() encountered an error (return code 1) while executing '. ./venv/bin/activate && nosetests -w None '
```
A more user-friendly error-message like `non-existent test suite: non_existent` would help.

**Comment by evansde77** [comment]
https://github.com/evansde77/cirrus/blob/develop/src/cirrus/test.py#L57 
https://github.com/evansde77/cirrus/blob/develop/src/cirrus/test.py#L78 
should both check to see if suite_conf is None and raise/log if it is.

**Comment by evansde77** [comment]
This was included in the 0.2.X releases
